Tests/TestingApps/PaywallsTester/PaywallsTester/Info.plist

```diff
@@ -2,6 +2,19 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>CFBundleURLTypes</key>
+	<array>
+		<dict>
+			<key>CFBundleTypeRole</key>
+			<string>Editor</string>
+			<key>CFBundleURLName</key>
+			<string>RevenueCat</string>
+			<key>CFBundleURLSchemes</key>
+			<array>
+				<string>rc-39976e83d0</string>
+			</array>
+		</dict>
+	</array>
```

Comment:
unsure what this is

Reply:
Ah right, this is to support custom schema deep links in iOS. Basically the `rc-39976e83d0` is the code for our test app... We could potentially keep it empty, but I feel having a default might make it easier?